pwiz_tools/Skyline/TestTutorial/GroupedStudiesTutorialTest.cs

```diff
@@ -202,19 +200,23 @@ protected override void DoTest()
                 OkDialog(peptideSettingsUI2, peptideSettingsUI2.OkDialog);
             }
 
-            RunUI(() => SkylineWindow.SaveDocument(GetOcRawTestPath("OC-study.sky")));
+            RunUI(() => SkylineWindow.SaveDocument(GetTestPath("OC-study.sky")));
 
             // Importing Data
             ImportResultsFiles(GetOcRawTestPath(), ExtAbWiff,
                 IsFullData ? "R201217" : "R201217_plasma_revision_A", true);
             WaitForCondition(5*60*1000, () =>
                 SkylineWindow.Document.Settings.HasResults && SkylineWindow.Document.Settings.MeasuredResults.IsLoaded);
+            WaitForDocumentLoaded();
+
             ImportResultsFiles(GetOcRawTestPath(), ExtAbWiff,
                 IsFullData ? "R201203" : "R201203_plasma_revision_F", true);
             WaitForCondition(2*60*1000, () =>
                 SkylineWindow.Document.Settings.HasResults && SkylineWindow.Document.Settings.MeasuredResults.IsLoaded);
+            WaitForDocumentLoaded();
 
             PauseForScreenShot();
+            LoadNewDocument(true); // Clean up

```

Comment:
Are you sure this is necessary? AbstractFunctionalTest.EndTest() does essentially this, I think with:

                // Release all resources by setting the document to something that
                // holds no file handles.
                var docNew = new SrmDocument(SrmSettingsList.GetDefault());
                // Try twice, because this operation can fail due to active background processing
                RunUI(() => Helpers.TryTwice(() => SkylineWindow.SwitchDocument(docNew, null)));

Reply:
Yeah, likely not required. I'll delete it.